SUMMARY.md

```diff
@@ -446,6 +447,7 @@
       - [HMAC Checklist](sw/device/lib/dif/dif_hmac.md)
       - [I2C Checklist](sw/device/lib/dif/dif_i2c.md)
       - [Key Manager Checklist](sw/device/lib/dif/dif_keymgr.md)
+      - [Key Manager (DPE) Checklist](sw/device/lib/dif/dif_keymgr_dpe.md)
```

Comment:
Not a big deal, but I think the headings above are "Key Manager" and "Key Manager DPE". Maybe drop the parens here?